samples/client/others/typescript/builds/enum-single-value/models/SingleValueEnum31.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * 
+ * No description provided (generated by Openapi Generator https://github.com/openapitools/openapi-generator)
+ *
+ * OpenAPI spec version: 1.0.0
+ * 
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+import { HttpFile } from '../http/http';
+
+export class SingleValueEnum31 {
+    'type': string;
```

Comment:
this should be:
```suggestion
    'type': 'this-is-my-only-value';
```

Reply:
fix in [9d36e99](https://github.com/OpenAPITools/openapi-generator/pull/19696/commits/9d36e993102d7573977dd0b62284005a1de287df)